terraform/table_terraform_module.go

```diff
@@ -0,0 +1,135 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/Checkmarx/kics/pkg/model"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+func tableTerraformModule(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "terraform_module",
+		Description: "Terraform module information.",
+		List: &plugin.ListConfig{
+			ParentHydrate: tfConfigList,
+			Hydrate:       listModules,
+			KeyColumns:    plugin.OptionalColumns([]string{"path"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Module name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "source",
```

Comment:
@rollwagen We now have a standard `source` column we add to tables, which is the full source code of the resource block. So if we wanted to add that same type of column, I think we'd add it as `source` (to keep it consistent with the other tables' columns), so we could rename this column to `module_source`.

Reply:
Thanks for pointing this out; makes sense & addressed in latest commit.